packages/browser-extension/src/scripts/dialog.js

```diff
@@ -205,9 +217,13 @@ async function submitButtonClickHandler(event) {
   const reason = Number.isNaN(reasonIndex) ? 'Unknown' : reasons[reasonIndex];
   const submitView = dialog?.getElementsByTagName('report-dialog-submit-view')[0];
   const userAgent = window.navigator.userAgent;
-
-  const issueUrl = await dispatch({ userAgent, reason, type: 'REPORT' });
-
+  const textareaValue = dialog?.querySelector('#explanation').value;
```

Comment:
After the changes above, this will become `const textAreaValue = dialog?.getElementsByTagName('report-dialog-text-area')[0];`.

Reply:
Please, after the previous task move it at the top of `const userAgent = ...`.